src/python/WMCore/JobSplitting/EventAwareLumiBased.py

```diff
@@ -169,9 +171,10 @@ def algorithm(self, *args, **kwargs):
                 # and it's only one lumi then ditch that lumi
                 timePerLumi = f['avgEvtsPerLumi'] * timePerEvent
                 if timePerLumi > jobTimeLimit and f['lumiCount'] == 1:
-                    failNextJob = True
-                    stopJob = True
```

Comment:
From what I see `stopJob` means the splitting for the current job is completed and we don't need to iterate over another file. So I think this change is Ok, but can you have another look just to be safer?

Reply:
Yes, this seems okay according to the unit tests (seems to do what I want, it actually doesn't if I  set `stopJob = True` only when `allowCreationFailure = True` )